core/templates/dev/head/pages/teach/Teach.ts

```diff
@@ -16,6 +16,8 @@
  * @fileoverview Controllers for the teach page.
  */
 
+require('pages/FooterDirective.ts');
```

Comment:
Why is this above separately? (I'm not sure how we organize .ts import in the directive/controller.)

Reply:
I saw it [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#adding-require-to-the-typescript-files-with-directive). Does that make sense?

Reply:
Thanks for the link, that's really helpful. 
In that case, I think the import in `pages/about/About.ts` file is incorrect, isn't it? (Earlier I was considering that as a correct order. :p)

Reply:
Thanks for pointing that out!
Done.